src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs

```diff
@@ -113,8 +113,11 @@ public void LoadBuildingBlocksFromTemplateToModule(Module module)
 
       public void RemoveModule(Module module)
       {
-         context.AddToHistory(new RemoveModuleCommand(module)
-            .Run(context));
+         var referringSimulations = Context.CurrentProject.SimulationsCreatedUsing(module);
```

Comment:
There was no test for module in use before. But to get it working, we need to sync the names.
Syncing the names is the bulk of the work here.